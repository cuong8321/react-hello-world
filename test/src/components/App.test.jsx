import './__lib__/material-ui-mocks.jsx'
import React from 'react'
import renderer from 'react-test-renderer'
import App from '../../../src/components/App.jsx'

test('Snapshot: components/App.jsx', () => {
  const init = () => 0.123456789
  const tree = renderer.create(<App init={init} />).toJSON()
  expect(tree).toMatchSnapshot()
})
