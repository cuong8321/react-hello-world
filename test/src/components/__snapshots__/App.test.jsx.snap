// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot: components/App.jsx 1`] = `
<mocked-component
  muiTheme={
    Object {
      "theme": undefined,
    }
  }
>
  <div>
    <mocked-component
      title="Hello, World!!"
    />
    <mocked-component
      zDepth={0}
    >
      <span
        className="label"
      >
        Random Number: 
      </span>
      <span
        className="value"
      >
        1F9ADD3739635F
      </span>
    </mocked-component>
  </div>
</mocked-component>
`;
